deploy/workloads/ping-latency-firemarshal.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
why is this file called 'ping-latency-firemarshal.json' and not just 'ping-latency.json'?

Reply:
the workload is actually not generated with firemarshal, it still relies on the old ping-latency.json to be built using the old gen-benchmark-rootfs.py 

but in theory, even when the workload is generated this way, you should be able to run it using ping-latency-firemarshal.json.

Reply:
alright, that's fine.